hls_vi/generate_stac_items.py

```diff
@@ -131,6 +131,8 @@ def process_common_metadata(item, granule):
     instrument = (
         granule.Platforms.Platform.Instruments.Instrument.ShortName.cdata.lower()
     )
+    # a simple check to exclude Sentinel-2 from the ShortName string for S30 granules
+    # and OLI from the short string for L30 granules.
```

Comment:
This doesn't make it clear as to why we're splitting on a space. Consider something like this:
```suggestion
    # For L30, the instrument is "OLI", but for S30, it is "Sentinel-2 MSI", we simply split
    # on spaces and grab the last element, so we get either "OLI" or "MSI".
```

In fact, I suggest we simplify by eliminating the if/else completely, and change the assignment that follows to this:

```python
    item.common_metadata.instruments = [instrument.split(" ")[-1]]
```